index.js

```diff
@@ -9,13 +9,27 @@ exports.prepare = function (extensions, filepath, cwd, nothrow) {
   var attempts = [];
   var err;
   var onlyErrors = false;
-  var ext = extension(filepath);
-  if (Object.keys(require.extensions).indexOf(ext) !== -1) {
+  var exts = extension(filepath);
+
+  var supportsExtension = Object.keys(require.extensions).indexOf(exts[0]) !== -1;
```

Comment:
but shouldn't we check every extension to see if we support it? e.g. what if a user names something .babel.coffee for some reason